super_calendar/models/super_calendar.py

```diff
@@ -26,7 +26,7 @@
 #
 ##############################################################################
 
```

Comment:
Could you please replace with short headers in all py files following: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py